modules/component/src/core/tick-scheduler.ts

```diff
@@ -27,10 +34,18 @@ export class AnimationFrameTickScheduler extends TickScheduler {
   schedule(): void {
     if (!this.isScheduled) {
       this.isScheduled = true;
-      requestAnimationFrame(() => {
-        this.appRef.tick();
-        this.isScheduled = false;
-      });
+
+      if (this.isServer) {
+        requestAnimationFrame(() => {
+          this.appRef.tick();
+          this.isScheduled = false;
+        });
+      } else {
+        setTimeout(() => {
+          this.appRef.tick();
+          this.isScheduled = false;
+        });
+      }
```

Comment:
There is no need to fully duplicate logic. Also, the condition has to be reversed - `requestAnimationFrame` should be used when `isServer` is `false`, not `true`:

1. Add a class member:

```ts
private readonly scheduleFn = this.isServer ? setTimeout : requestAnimationFrame;
```

2. Then use `scheduleFn` within the `schedule` method:

```suggestion
      scheduleFn(() => {
        this.appRef.tick();
        this.isScheduled = false;
      });
```